articles/client-auth/v2/mobile-desktop.md

```diff
@@ -0,0 +1,394 @@
+    "text": "{\"grant_type\":\"authorization_code\",\"client_id\": \"${account.clientId}\",\"code_verifier\": \"YOUR_GENERATED_CODE_VERIFIER\",\"code\": \"YOUR_AUTHORIZATION_CODE\",\"redirect_uri\": \"https://${account.namespace}/mobile\", }"
+  }
+}
+```
+
+Please refer to [this page](/api-auth/tutorials/authorization-code-grant-pkce#4-exchange-the-authorization-code-for-an-access-token) for additional details on the ID Token request parameters.
+
+If all goes well, you'll receive an HTTP 200 response with the following payload:
+
+```json
+{
+  "access_token":"eyJz93a...k4laUWw",
+  "refresh_token":"GEbRxBN...edjnXbL",
+  "id_token":"eyJ0XAi...4faeEoQ",
+  "token_type":"Bearer",
+  "expires_in":86400
+}
+```
+
+:::panel-info Access Tokens
+You can use `access_token` to call the [Authentication API's `/userinfo` endpoint](/api/authentication#get-user-info).
+:::
+
+### Keep the User Logged In
+
+Auth0 assists in authenticating a user, but your application must keep track of whether or not a user is logged in. You can keep a global variable or a singleton object inside your application to do this. You can also use this object to store information about the user (such as name and profile image) so that you can deliver a personalized user experience within your application.
+
+## The `id_token`
+
+Once you've decoded the Access Token, you can extract user information from the `id_token` payload. The JSON payload contains the user claims (attributes), as well as metadata, and it will look something like this:
+
+```json
+{
+  "name": "John Smith",
+  "email": "jsmith@example.com",
+  "picture": "https://example.com/profile-pic.png",
+  "iss": "https://auth0user.auth0.com/",
+  "sub": "auth0|581...",
+  "aud": "xvt...",
+  "exp": 1478113129,
+  "iat": 1478077129
+}
+```
+
+For additional details, please see our docs [on the ID Token and its claims](/tokens/id-token#id-token-payload).
+
+## Example Use Cases
+
+This section covers use cases that illustrate the authentication process using PKCE.
+
+### Request the User's Name and Profile Picture
+
+In addition to the usual authentication, this example shows how you can request additional user details.
+
+We assume that your app is capable of generating the appropriate [code verifier](#code-verifier) and [code challenge](#code-challenge).
+
+To return the user's `name` and `picture`, add the appropriate scopes to your call to the `/authorize` endpoint. Therefore, the initial authorization URL is as follows:
+
+```text
+https://${account.namespace}/authorize?
+    scope=openid%20name%20picture&
+    response_type=code&
+    client_id=${account.clientId}&
+    code_challenge=CODE_CHALLENGE&
+    code_challenge_method=S256&
+    redirect_uri=${account.namespace}/mobile
+```
+
+After the user provides submits their credentials, your app receives an HTTP 302 response with a URL containing the authorization code at the end: `https://YOUR_APP/callback?code=AUTHORIZATION_CODE`
```

Comment:
Change the `https://YOUR_APP/callback?code=AUTHORIZATION_CODE` to `${account.namespace}/mobile?code=AUTHORIZATION_CODE`